### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by Mindfulplays**
Moved from https://github.com/MonoGame/MonoGame/pull/7950

@SimonDarksideJ @tomspilman

### CODE REVIEW

**Comment by harry-cpp** [comment]
@Mindfulplays Its gonna start getting a bit difficult to check your PRs as you are already at 43 commits due to how you are resolving your merge conflicts. Try using the rebase command as I mentioned before, in this case for example it would be:
```sh
git checkout doc-update # switch branch
git fetch upstream # fetch `upstream` repo changes
git rebase upstream/develop # merge upstream repo commits before your commits instead of after them
git push -u origin doc-update --force # push changes upstream, needs force as its inserting commits before your newest one so its rewriting history in a way
```
(assuming you are using `origin` to represent `Mindfulplays/MonoGame` and `upstream` to represent `MonoGame/MonoGame`)

**Comment by harry-cpp** [comment]
Actually looking at it, you have pushed some changes than reverted them by making new commits, well, this will be a little more tricky to fully cleanup. After you've cleaned up the merge commits issues with the above comment, you can delete the commits you don't need by doing:
```
git rebase -i HEAD~5 # replace 5 with how many commits you wanna modify
```

You'll get an interactive prompt in the default text editor where you just need to modify the first letter before the commit with what action you want to do. All available actions are explained at the bottom of the file. Don't modify anything here except the first letters before the commits, if you wanna reword the commit message and select that as an action, you'll get the prompt for it after this one is done.

**Comment by Mindfulplays** [comment]
> Actually looking at it, you have pushed some changes than reverted them by making new commits, well, this will be a little more tricky to fully cleanup. After you've cleaned up the merge commits issues with the above comment, you can delete the commits you don't need by doing:
> 
> ```
> git rebase -i HEAD~5 # replace 5 with how many commits you wanna modify
> ```
> 
> You'll get an interactive prompt in the default text editor where you just need to modify the first letter before the commit with what action you want to do. All available actions are explained at the bottom of the file. Don't modify anything here except the first letters before the commits, ie. if you wanna reword the commit message and select that as an action, you'll get the prompt for it after this one is done.

Thanks for the detailed steps, I will do this for all my PRs (including the current ones) next week. Please bear with me until then.

File: Tests/README.md [hunk 1]
```diff
@@ -144,12 +166,12 @@ the test fixture is already set up):
...
 
-- You must run the -x86 versions of NUnit, because XNA won't work with
+- You must run the **-x86** versions of NUnit, because XNA won't work with
   the 64-bit versions.
 - You must disable shadow copying because having it enabled makes it
   impossible for the ContentManager to find any assets.
-  - GUI: Tools > Settings > Test Loader > Advanced
-  - CLI: /noshadow
+  - ```GUI: Tools > Settings > Test Loader > Advanced```
+  - ```CLI: /noshadow```
```
Comment (harry-cpp) [inline Tests/README.md:174]: Use 1 quote for single code line like this, not 3 of em, same for a lot of cases above.

Comment (harry-cpp) [reply Tests/README.md:174]: Actually the entire file is using triple ones, I see XD

**Pull request merged**